core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/IntelliJPaths.scala

```diff
@@ -14,7 +14,7 @@ final case class IntelliJPaths(
     userPrefs: Path
 ) {
   val bin: Path = root.resolve("bin")
-  val bundledPlugins = root.resolve("plugins")
+  val bundledPlugins: Path = root.resolve("plugins")
```

Comment:
It's now only used in

<img width="629" alt="image" src="https://github.com/VirtusLab/ide-probe/assets/3383210/6129de16-de73-494c-a1c5-f1a8c496f2cb">